.github/workflows/assets.yml

```diff
@@ -1,26 +1,36 @@
-name: Compile assets
+name: Compile dist
 
 on:
-  push:
-    paths:
-      - 'www/assets/**'
-  pull_request:
+  pull_request_review:
+    types:
+      - submitted
     paths:
       - 'www/assets/**'
 
 jobs:
-  assets:
+  build:
+    if: github.event.review.state == 'approved'
     runs-on: ubuntu-16.04
+
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout getkirby.com
+        uses: actions/checkout@v2
         with:
           ref: ${{ github.head_ref }}
-      - uses: actions/setup-node@v2
+
+      - name: Setup node environment
+        uses: actions/setup-node@v2
         with:
           node-version: 12
-      - run: npm install
-      - run: npm run build
-      - uses: stefanzweifel/git-auto-commit-action@v4
+
+      - name: Install npm
+        run: npm install
```

Comment:
I'd recommend using `npm ci` instead, which can use the global cache in a better way. See the workflow in the Kirby repo.